.pre-commit-config.yaml

```diff
@@ -10,12 +10,7 @@ repos:
     hooks:
     -   id: seed-isort-config
         args: ['--application-directories=./src']
--   repo: https://github.com/pre-commit/mirrors-isort
-    rev: v4.3.21  # pick the isort version you'd like to use from https://github.com/pre-commit/mirrors-isort/releases
-    hooks:
-    -   id: isort
-        additional_dependencies:
-          - toml
+
```

Comment:
had problem to commit `websocket_client_test.py` with this but should try to put this back